.github/workflows/release.yaml

```diff
@@ -0,0 +1,23 @@
+name: Release
+
+on:
+  release:
+    types: [published]
```

Comment:
Published seems best <https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release> 👍🏻